packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -199,6 +204,8 @@ const App = (props) => {
     const updateBasket = useShopperBasketsMutation('updateBasket')
     const updateCustomerForBasket = useShopperBasketsMutation('updateCustomerForBasket')
 
+    const isBlocked = useRouting()
```

Comment:
It's hard to know what this hook is doing based on its name `useRouting`. Ideally for readability we are able to come up with a name that is a little more self descriptive.